### PR TITLE
Update llama.cpp gguf to latest

### PR DESCRIPTION
**Pull request by BruceMacD**
- Update 0001-remove-warm-up-logging.patch

There have been some bug fixes and improvements, updating the llama.cpp gguf runner to latest to get these in our next release.

### CODE REVIEW

File: llm/llama.cpp/patches/0001-remove-warm-up-logging.patch
Comment (mxyng) [inline llm/llama.cpp/patches/0001-remove-warm-up-logging.patch:1]: Since the log line is gone, you can delete patch file

**Review by mxyng — COMMENTED** [review]
Did the metal performance regression get fixed

File: llm/llama.cpp/patches/0001-remove-warm-up-logging.patch
Comment (BruceMacD) [reply llm/llama.cpp/patches/0001-remove-warm-up-logging.patch:1]: looks like its still there, just moved
https://github.com/ggerganov/llama.cpp/blob/acec9eaaa93315711c11d15afa8d245d164b7cff/common/common.cpp#L845

**Comment by BruceMacD** [comment]
Which metal regression? There's quite a few different ones open/closed at the moment.
https://github.com/ggerganov/llama.cpp/issues?q=is%3Aissue+is%3Aopen+metal

File: llm/llama.cpp/patches/0001-remove-warm-up-logging.patch
Comment (mxyng) [reply llm/llama.cpp/patches/0001-remove-warm-up-logging.patch:1]: In that case the patch file should be updated

**Comment by mxyng** [comment]
This should be good to merge. The aforementioned regression has limited impact on ollama

File: llm/llama.cpp/patches/0001-remove-warm-up-logging.patch
Comment (mxyng) [reply llm/llama.cpp/patches/0001-remove-warm-up-logging.patch:1]: My bad. This has been updated. Reading a diff of a diff is not easy

**Review by mxyng — COMMENTED** [review]
Needs to be updated again but overall this is ready for review

**Review by mxyng — APPROVED** [review]

**Pull request merged**